packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -260,6 +261,10 @@ class PaginatedDataTable extends StatefulWidget {
   /// and no affordance will be provided to change the value.
   final ValueChanged<int?>? onRowsPerPageChanged;
 
+  /// Used as a string replacement for Rows Per Title text in Pagination Footer Section
+  /// Defaults to [localizations.rowsPerPageTitle]
+  final String? rowsPerPageTitle;
```

Comment:
```suggestion
  /// When provided, this string replaces the rows per title in the footer.
  ///
  /// Defaults to [MaterialLocalizations.rowsPerPageTitle].
  final String? rowsPerPageTitle;
```
`localizations.rowsPerPageTitle` reference is still invalid as it is looking for `MaterialLocalizations` itself, not the variable.